pool.go

```diff
@@ -50,6 +60,11 @@ func (p *pool) Store(v wire) {
 	p.cond.L.Lock()
 	if !p.down && v.Error() == nil {
 		p.list = append(p.list, v)
+
+		if p.idleConnTTL != nil {
+			p.removeIdleConns()
```

Comment:
Hi @veirfuna,

I guess calling `.removeIdleConns()` alone in the `.Store()` can't remove idle connections. For example, if `.Store()` is not called, then idle connections are still there. You probably need to put `.removeIdleConns()` into a ticker.

and thanks for your contribution!

Reply:
Hi @rueian,

My first attempts were to create a goroutine that is observe idle connections and remove them. Do you see any problems in such solution? The only idea to not use goroutine was to add `.removeIdleConns()` to `.Store()`

Reply:
I think we have no choice but to use an additional goroutine. We can use time.AfterFunc and it is actually an additional goroutine. We can schedule the cleanup function only when p.size > minIdle.